### PR TITLE
Fix diagram mistakes

### PR DESCRIPTION
**Pull request by theodorekwok**
- Fix some diagram mistakes according to the dg review
- Updated diagrams based on recent prs
https://github.com/theodorekwok/tp/blob/theodorekwok-fix-diagram-mistakes/docs/DeveloperGuide.md

### CODE REVIEW

**Review by KVignesh122 — COMMENTED** [review]
Excellent job. 2 other diagrams need small rectifications. Comments posted separately.

**Comment by KVignesh122** [comment]
As suggested by Rafael:
image

These arrows pointing to abstract instrument should be solid lines to denote inheritance rather than dependency.

**Comment by KVignesh122** [comment]
image

The constructor gap glitch is still present in this diagram, under DoneCommand.

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -47,14 +51,15 @@ The main parent class in `console` package is the `InputParser` class which is d
...
-<img src="images/ConsoleDiagram.png" width="550"/>
+<img src="images/ConsoleDiagram.png" width="1040"/>
 
 How the `InputParser` class works:
 1. When the user enters a command along with the relevant parameters if any, the
    `getCommandComponents()` method in `InputParser` separates the user's command by spaces to return a string array.
 2. The command is then determined by using the `filterByCommandType()` method which would return the corresponding
    command type. Examples of different command types are `AddInstrumentCommand`, `DeleteCommand`, `ListCommand` etc.
 
+#### Design considerations for parsing inputs for add functionality
```
[the remark this answers is not in the document]
Comment (theodorekwok) [reply docs/DeveloperGuide.md:62]: Ohh good point, i just wanted to make some bold header that was smaller than the main header

**Comment by theodorekwok** [comment]
> image
> 
> The constructor gap glitch is still present in this diagram, under DoneCommand.

**Comment by williamlamjy** [comment]
i think can merge this in first i can continue the edits on the diagrams on my storage implementation pr

**Pull request merged**